.github/workflows/deploy_staging.yml

```diff
@@ -0,0 +1,127 @@
+name: Deploy Staging
+
+on:
+  # Only trigger this workflow on push action in develop branch
+  push:
+    branches:
+      - develop
+jobs:
+  deploy_staging:
```

Comment:
I think this should rename to `run_test_and_deploy_staging` for more understanding. 🙏🏻